client/src/app/map/map.component.ts

```diff
@@ -131,11 +132,27 @@ export class MapComponent implements OnInit {
         for (var i = 0; i < this.preparednessLayers.length; i++){
             var newSource = new ol.source.TileWMS({
                 params: {
-                    'LAYERS': this.preparednessLayers[i].layerName,
+                    'LAYERS': 'dmis:' + this.preparednessLayers[i].layerName,
```

Comment:
Need to decide between specifying the workspace here and changing the target url in the proxy??